File: lang/php/test/vac.fun/util/TestRunnerTest.php
<?php
    require_once "vac.fun/util/TestRunner.php";

    use vac\fun\util\TestRunner;

    function testParseTest(): void {
        for ($i = 0; $i < 10; $i++) {
            TestRunner\parseTest(($i & 1) != 0);
        }
    }

    if (count(debug_backtrace()) == 0) {
        testParseTest();
    }
?>

File: lang/c++/test/vac.fun/util/TestRunnerTest.cpp
#include <vac.fun/util/TestRunner.hpp>

using namespace std;

void testParseTest() {
    for (int i = 0; i < 10; i++) {
        TestRunner::parseTest((i & 1) != 0);
    }
}

int main() {
    testParseTest();
}
